Python/Product/PythonTools/PythonTools/Commands/ImportCoverageCommand.cs

```diff
@@ -37,7 +40,7 @@ internal sealed class ImportCoverageCommand : Command {
             
         }
 
-        public override void DoCommand(object sender, EventArgs args) {
+        public override async void DoCommand(object sender, EventArgs args) {
```

Comment:
Do we have async commands?

Reply:
Not yet. Can we implement them?

Reply:
I filed https://github.com/Microsoft/PTVS/issues/3942 for this.
For now you could just extract it in a Task returning method and call DoNotWait(). At least that way @AlexanderSher 's special `DoNotWait` injection for detecting non terminating task will work.